modules/Api/Controllers/AuthController.php

```diff
@@ -39,6 +39,43 @@ class AuthController extends ApiController
      * Sign in action
```

Comment:
Only the method description needed to be moved to the top not a param definitions